zipkin-cassandra/src/main/scala/com/twitter/zipkin/cassandra/CassandraSpanStoreFactory.scala

```diff
@@ -58,9 +58,11 @@ trait CassandraSpanStoreFactory {self: App =>
   }
 
   def addContactPoint(builder: Cluster.Builder): Cluster.Builder = {
-    val contactPoint = HostAndPort.fromString(cassandraDest())
+    val contactPoints = cassandraDest().split(",").map(HostAndPort.fromString)
```

Comment:
this is a little complex.. why not just split on comma, then pass that directly to `.addContactPoints(contactPoints:_*)`